OKImageDownloader/Classes/Extensions/UIImageView+ImageDownloader.swift

```diff
@@ -27,7 +27,9 @@ public extension UIImageView {
         }
     }
     
-    public func downloadImage(with url: URL, imageDownloader: ImageDownloading = ImageDownloader.shared, completionHandler: ImageDownloader.CompletionHandler?) {
+    func downloadImage(with url: URL, imageDownloader: ImageDownloading = ImageDownloader.shared, completionHandler: ImageDownloader.CompletionHandler?) {
+        cancelDownloadImage()
```

Comment:
@joel-okcupid This should resolve a reloadData() or pagination call where we download a new image on an in memory UIImageView that already has an image download in progress.